admin/system/login-proses.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+    include("connection.php");
+    session_start();
+
+    $username = $_POST["username"];
+    $password = $_POST["password"];
```

Comment:
Codacy found an issue: [Direct use of $_POST Superglobal detected.](https://app.codacy.com/manual/anamnafiul99/pindad-clone/pullRequest?prid=5762293)